roofit/hs3/src/JSONFactories_RooFitCore.cxx

```diff
@@ -96,39 +167,148 @@ class RooSimultaneousFactory : public RooJSONFactoryWSTool::Importer {
       if (!p.has_child("channels")) {
          RooJSONFactoryWSTool::error("no channel components of '" + name + "'");
       }
-      tool->importPdfs(p["channels"]);
       std::map<std::string, RooAbsPdf *> components;
-      RooCategory cat("channelCat", "channelCat");
+      std::string indexname(p["index"].val());
+      RooCategory cat(indexname.c_str(), indexname.c_str());
       for (const auto &comp : p["channels"].children()) {
          std::string catname(RooJSONFactoryWSTool::name(comp));
+         RooMsgService::instance().log(tool, RooFit::INFO, RooFit::IO) << "importing category " << catname << std::endl;
```

Comment:
```suggestion
         tool->log(RooFit::INFO) << "importing category " << catname << std::endl;
```
With my suggested log function, it would look like this